README.md

```diff
@@ -1522,6 +1522,7 @@ See also [Are we web yet?](http://www.arewewebyet.org) and [Rust web framework c
   * [bitwyre/websocket_core](https://github.com/bitwyre/websocket_core) — Websocket generic server library for periodic message broadcast. [<img src="https://github.com/bitwyre/websocket_core/workflows/Build/badge.svg?branch=master">](https://github.com/bitwyre/websocket_core/actions)
 * Miscellaneous
   * [cargonauts](https://github.com/cargonauts-rs/cargonauts) — A web framework intended for building maintainable, well-factored web apps.
+  * [osohq/oso](https://github.com/osohq/oso) [[oso](https://crates.io/crates/oso)] - A policy engine for authorization that's embedded in your application.
```

Comment:
Can you please add the build badge for the Github actions work to this?

Reply:
Thanks for the suggestion. Added.